my-app/src/pages/register.tsx

```diff
@@ -12,45 +11,53 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 const theme = createTheme();
 
 const Register = () => {
-
     return (
         <ThemeProvider theme={theme}>
             <Grid container component="main" style={{ height: '100vh' }}>
-            <CssBaseline />
-                <Grid
-                    item
+                <CssBaseline />
+
+                <Grid item
                     xs={false}
```

Comment:
it may be helpful to put these styling into their own CSS file. Not important for now but it will be annoying to refactor in the future